assignments/closure.js

```diff
@@ -1,10 +1,17 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+// function myFunction() {
+//   const name = "Nicholas";
+//   function greetMe() {
+//     console.log(`Hello ${name}`);
+//   }
+//   return greetMe;
+// }
+// let func = myFunction();
+// func();
```

Comment:
Nice! If you don't like having to assign your function twice, you can call your closure inside the parent function instead of returning it.
```suggestion
function myFunction() {
  const name = "Nicholas";
  function greetMe() {
    console.log(`Hello ${name}`);
  }
  greetMe();
}
myFunction();
```